src/components/eft/ShortNameRefund.vue

```diff
@@ -298,7 +295,7 @@ export default defineComponent({
 
     function initiateRefund () {
       const enableRefundByCheque: boolean = LaunchDarklyService.getFlag(LDFlags.EnableEFTRefundByCheque, false)
-      const routeName = enableRefundByCheque ? 'shortnamerefundselection' : 'shortnamerefund'
+      const routeName = enableRefundByCheque ? 'shortnamerefundselection' : 'shortnamerefunddirectdeposit'
```

Comment:
worth kicking these routes to an enum? that way if it ever gets refactored, only 1 spot changes?